solutions/usaco-991.mdx

```diff
@@ -70,36 +70,37 @@ public class Loan {
   
 <PySection>
 ```
-# our checking function in the binary search
-def isOk(numGallons, withinDays, atLeast, checkNumber):
-  	g = 0
-  	while (withinDays > 0 and g < numGallons):
-  	    y = (numGallons - g) // checkNumber
-  	    
-  	    if (y < atLeast):
-  	        leftover = ((numGallons - g) + (atLeast - 1)) // atLeast
-  	        return leftover <= withinDays
-  	        
-  	    maxMatch = numGallons - (checkNumber * y)
-  	    numDays = (maxMatch - g) // y + 1
-  	    
-  	    if (numDays > withinDays): # take a maximum
-  	        numDays = withinDays
-  	    g += y * numDays # update values
-  	    withinDays -= numDays
-  
-  	return g >= numGallons
+class Loan:
```

Comment:
why put this in a class?

Reply:
added class to make the code similar to Java Code which is present.

Reply:
python != java, so i'd take it out